commit 26530269d07cb46e42253997992f2e58408157a7
Author: Matt Flax <flatmax@flatmax.org>
Date:   Fri Sep 28 15:13:28 2018 +1000

    ASoC: cs4265: Add a MIC pre. route (#2696)
    
    Commit  b0ef5011b981ece1fde8063243a56d3038b87adb upstream.
    
    The cs4265 driver is missing a microphone preamp enable.
    This patch enables/disables the microphone preamp when mic
    selection is made using the kcontrol.
    
    Signed-off-by: Matt Flax <flatmax@flatmax.org>
    Reviewed-by: Charles Keepax <ckeepax@opensource.cirrus.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/codecs/cs4265.c b/sound/soc/codecs/cs4265.c
index 6d0843386948..198ae46ebe9b 100644
--- a/sound/soc/codecs/cs4265.c
+++ b/sound/soc/codecs/cs4265.c
@@ -222,10 +222,11 @@ static const struct snd_soc_dapm_route cs4265_audio_map[] = {
 	{"LINEOUTR", NULL, "DAC"},
 	{"SPDIFOUT", NULL, "SPDIF"},
 
+	{"Pre-amp MIC", NULL, "MICL"},
+	{"Pre-amp MIC", NULL, "MICR"},
+	{"ADC Mux", "MIC", "Pre-amp MIC"},
 	{"ADC Mux", "LINEIN", "LINEINL"},
 	{"ADC Mux", "LINEIN", "LINEINR"},
-	{"ADC Mux", "MIC", "MICL"},
-	{"ADC Mux", "MIC", "MICR"},
 	{"ADC", NULL, "ADC Mux"},
 	{"DOUT", NULL, "ADC"},
 	{"DAI1 Capture", NULL, "DOUT"},
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1272_26530269d07cb46e42253997992f2e58408157a7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1272_26530269d07cb46e42253997992f2e58408157a7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
